src/less/details/ds6/details.less

```diff
@@ -0,0 +1,36 @@
+@import "../../less/ds6/variables.less";
+
+details:not([open]) {
+    .show-less {
```

Comment:
Be sure to use proper BEM syntax for child elements.

Reply:
`details__summary--showmore`?